client/web/src/enterprise/batches/settings/BatchChangesSettingsArea.tsx

```diff
@@ -14,6 +14,10 @@ export interface BatchChangesSettingsAreaProps extends Pick<RouteComponentProps,
 export const BatchChangesSettingsArea: React.FunctionComponent<BatchChangesSettingsAreaProps> = props => (
     <div className="web-content test-batches-settings-page">
         <PageTitle title="Batch changes settings" />
-        <CodeHostConnections userID={props.user.id} {...props} />
+        <CodeHostConnections
+            headerLine={<p>Add authentication tokens to enable batch changes changeset creation on your code hosts.</p>}
```

Comment:
Something about "authentication tokens" feels off to me. Not sure what or whether I'm seeing ghosts.

But how about "access tokens"?

```suggestion
            headerLine={<p>Add access tokens to enable batch changes changeset creation on your code hosts.</p>}
```

Or "credentials"?

Reply:
Nevermind. I just saw that this is the wording we already use for the user settings page too.

But I'm still going to leave this comment here because I think the 'code host tokens' wording is a bit weird.